docs/Examples.md

```diff
@@ -87,6 +87,183 @@ $graphServiceClient = GraphServiceClient::createWithRequestAdapter($requestAdapt
 
 ```
 
+## Access token management
+Using the `TokenRequestContext`, an instance of the `GraphServiceClient` requests access tokens and refresh tokens.
+The tokens are stored by default in an in-memory cache so that future requests using the same instance of the `GraphServiceClient` can re-use the previously acquired tokens.
```

Comment:
Having a little bit of context on "how it's stored, how it's secured, etc." would be helpful. These tokens are important and should stay secure at all times.

Reply:
Added more info about this